packages/playwright-core/src/client/input.ts

```diff
@@ -89,4 +89,8 @@ export class Touchscreen implements api.Touchscreen {
   async tap(x: number, y: number) {
     await this._page._channel.touchscreenTap({ x, y });
   }
+
+  async touch(type: 'touchstart'|'touchmove'|'touchend'|'touchcancel', touchPoints: { x: number, y: number, id?: number }[]) {
```

Comment:
Looking at it, I prefer separate page.touchscreen.touchStart(points), touchEnd, touchMove, touchCancel. Maybe we are brave and do `page.touchscreen.start`

Reply:
Let's postpone this to the API review, for now we have bigger fish to fry.